lib/mandrill_dm/message.rb

```diff
@@ -65,6 +65,10 @@ def merge_language
       return_string_value(:merge_language)
     end
 
+    def merge_vars
+      mail[:merge_vars] ? mail[:merge_vars].instance_variable_get('@value') : nil
```

Comment:
Gotcha. Yes, `eval` is dangerous. That is worthy of a comment in the code to explain that.